setup.cfg

```diff
@@ -57,36 +58,36 @@ package_dir =
 install_requires =
     pip >= 22
     setuptools >= 60
-    wheel ~= 0.34
+    wheel ~= 0.37
     cookiecutter ~= 2.1
     tomli ~= 2.0; python_version <= "3.10"
-    importlib_metadata ~= 4.4; python_version <= "3.9"
-    requests ~= 2.22
+    importlib_metadata >= 4.4; python_version <= "3.9"
```

Comment:
This has been deliberately made a minimum version pin; importlib_metadata will track the implementation in the current version of Python, so we should ensure that we're tracking that version.